debian/changelog

```diff
@@ -1,4 +1,63 @@
-ubuntu-advantage-tools (27.1) impish; urgency=medium
+ubuntu-advantage-tools (27.2) impish; urgency=medium
+
+  * d/control:
+    - add comments to explain complex build-depends
+    - add version requirement to distro-info (LP: #1932028)
+  * New upstream release 27.2 (LP: #1934902):
```

Comment:
Since we are releasing to impish here, I don't think we need the SRU bug reference here

Reply:
Good point!